### PR TITLE
Submitted Polish translation

### PR DESCRIPTION
**Pull request by Ste74**
Our user send a translation 
https://forum.manjaro.org/t/gnome-editing-gno-menu-entries/14195/7

### CODE REVIEW

**Comment by The-Panacea-Projects** [comment]
Just merged the Polish translation file. Sorry for the delay but my schedule has been hectic lately.

**Comment by Ste74** [comment]
Don't worry 😄

File: po/pl_PL.po [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+#: extension.js
+
+msgid "Gno-Menu: Search Firefox bookmarks disabled"
+msgstr "Gno-Menu: Przeszukiwanie zakładek Firefox'a wyłączone"
+
+msgid "If you want to search Firefox bookmarks, you must install the required pacakages: libgir1.2-gda-5.0 [Ubuntu] or libgda-sqlite [Fedora]"
+msgstr "Jeśli chcesz przeszukiwać zakładki Firefox'a, musisz zainstalować paczkę: libgir1.2-gda-5.0 [Ubuntu] lub libgda-sqlite [Fedora]"
+
+msgid "Gno-Menu: Search Midori bookmarks disabled"
+msgstr "Gno-Menu: MPrzeszukiwanie zakładek Midori wyłączone"
```
Comment (edskeye) [inline po/pl_PL.po]: Hi, i'm the one who translated it. There's mistake in line #27.
It should be "Gno-Menu: Przeszukiwanie zakładek Midori wyłączone"
Without M before "Przeszukiwanie" - my mistake. Sorry for doing it non GitHub way - i'm just a graphic designer, not a developer ;)

**Comment by edskeye** [comment]
Hi, i'm the guy from Manjaro Forum who made translation. What i see is that you've used a different translation from pkomur. There are some phrases missing, so i made new one.
Sorry for non-GitHub way (i'm just a graphic designer, not a dev) but you can find it here:
https://github.com/edskeye/Gnomenu/blob/master/po/pl.po

**Comment by The-Panacea-Projects** [comment]
@edskeye, Sorry I didn't realize there were two different authors involved. After doing a comparison, I noticed there is quite a bit of difference between the two pl.po files. Are you saying that https://github.com/edskeye/Gnomenu/blob/master/po/pl.po should replace the currrent pl.po?

**Comment by edskeye** [comment]
@The-Panacea-Projects 
Hey, yes it's updated version with few more phrases translated (in config menu).

ex.

> msgid "Hide Shortcuts Panel"
msgstr "Ukryj panel skrótów"

I've checked it today as there were something missing so now it's updated.

**Pull request closed without merging**